hnn_core/basket.py

```diff
@@ -36,51 +47,16 @@ def __set_props(self, cell_name, pos):
             'name': cell_name,
         }
 
-    # For all synapses, section location 'secloc' is being explicitly supplied
-    # for clarity, even though they are (right now) always 0.5.
-    # Might change in future
-    # creates a RECEIVING inhibitory synapse at secloc
-    def syn_gabaa_create(self, secloc):
-        """Create gabaa receiving synapse."""
-        return self.syn_create(secloc, e=-80, tau1=0.5, tau2=5.)
-
-    def syn_ampa_create(self, secloc):
-        """Create ampa receiving synapse at secloc."""
-        return self.syn_create(secloc, e=0., tau1=0.5, tau2=5.)
-
-    # creates a RECEIVING nmda synapse at secloc
-    # this is a pretty fast NMDA, no?
-    def syn_nmda_create(self, secloc):
-        """Create nmda receiving synapse."""
-        return self.syn_create(secloc, e=0., tau1=1., tau2=20.)
-
     # creation of synapses
     def _synapse_create(self):
         # creates synapses onto this cell
-        self.soma_ampa = self.syn_ampa_create(self.soma(0.5))
-        self.soma_gabaa = self.syn_gabaa_create(self.soma(0.5))
-        self.soma_nmda = self.syn_nmda_create(self.soma(0.5))
-
-
-class L2Basket(BasketSingle):
-    """Class for layer 2 basket cells."""
-
-    def __init__(self, gid=-1, pos=-1):
-        # BasketSingle.__init__(self, pos, L, diam, Ra, cm)
-        # Note: Basket cell properties set in BasketSingle())
-        BasketSingle.__init__(self, gid, pos, 'L2Basket')
-        self.celltype = 'L2_basket'
-
-        self._synapse_create()
-        self._biophysics()
-
-    # par connect between all presynaptic cells
-    # no connections from L5Pyr or L5Basket to L2Baskets
-    def parconnect(self, gid, gid_dict, pos_dict, p):
-        self._connect(gid, gid_dict, pos_dict, p, 'L2_pyramidal', 'L2Pyr',
-                      postsyns=[self.soma_ampa])
-        self._connect(gid, gid_dict, pos_dict, p, 'L2_basket', 'L2Basket',
-                      lamtha=20., postsyns=[self.soma_gabaa])
+        self.synapses['soma_ampa'] = self.syn_create(
+            self.soma(0.5), e=0., tau1=0.5, tau2=5.)
+        self.synapses['soma_gabaa'] = self.syn_create(
+            self.soma(0.5), e=-80, tau1=0.5, tau2=5.)
+        # this is a pretty fast NMDA, no?
```

Comment:
Should this be clarified rather than having a comment buried in the code?

Reply:
umm ... what would you like me to clarify here? I actually just copy-pasted the old comment

Reply:
Comments that ask unanswered questions aren't that useful. Who were you asking in the first place? Are you comparing to other NMDA time constants somewhere?

https://github.com/jonescompneurolab/hnn-core/commit/41c8774ccd4ff56bb4338a70a1489fa65c9a2dd7#diff-faaeadb2c140c4e4e34881bb5f31ae87R52

Reply:
It's not my comment, cruft from old code: https://github.com/jonescompneurolab/hnn-core/commit/41c8774ccd4ff56bb4338a70a1489fa65c9a2dd7#diff-8a21b8909c8d5e3dc73c03688313de0eL351 :) I can remove it if you want to.

Reply:
Basically my strategy was to trash anything that was redundant with the code or other comments. But this one seemed a bit unique ...

Reply:
Oh, I see. Makes complete sense why it was copied forward. However, with all that effort spent on where the comment came from, I verified that the time constant is correct. These are the values have been used since at least 2009. See top of page 5: 
https://journals.physiology.org/doi/pdf/10.1152/jn.00535.2009

This comment can be removed.

Reply:
awesome, thank you so much for digging into this. I removed the comment in the last commit!